src/Util/InstallationUtil.php

```diff
@@ -287,4 +302,19 @@ private static function assertIpIsValid(string $ip)
             throw new BunqException(self::ERROR_INVALID_IP_ADDRESS, [$ip]);
         }
     }
+
+    /**
+     * @param string $deviceDescription
+     *
+     * @return bool
+     * @throws BunqException
+     */
+    public static function assertDeviceDescriptionIsValid(string $deviceDescription): bool
```

Comment:
I keep forgetting about this,.. do we have typehints for `string` in SDK?

Reply:
Yea, in the SDK we do not suffer from auto casting and therefore we indeed type hint strings.

Reply:
Ack.